mix.exs

```diff
@@ -1,4 +1,4 @@
-defmodule Benchfella.Mixfile do
+    defmodule Benchfella.Mixfile do
```

Comment:
I would humbly advice to run `mix format mix.exs`, commit and `git rebase -i`, then force-push into your PR branch to fix this excessive indentation.

Reply:
OMG how did that happen :(

Reply:
fix coming up